app/bundles/CoreBundle/Assets/css/app.css

```diff
@@ -5787,6 +5739,390 @@ lesshat-selector { -lh-property: 0 ;
 .gap-0 {
   gap: 0px !important;
 }
+@media (min-width: 768px) {
+  .fd-row-sm {
+    flex-direction: row;
+  }
+  .fd-row-reverse-sm {
+    flex-direction: row-reverse;
+  }
+  .fd-column-sm {
+    flex-direction: column;
+  }
```

Comment:
I like the change but I have a question about all these new CSS classes added in this PR. What are they for? How are they related to the user dropdown change? I don't understand the meaning from the naming and I'm afraid that not many people will be able to use them in the HTML. Can those be documented somehow?

Reply:
All new PRs that run `grunt compile-less` are generating these changes in app.css as a consequence of the new utilitities (already merged)
I believe it was a bug when running the command where not all changes in .less came into the .css file in that PR (or poor conflict management of another pull request that came after)

so we're safe to merge any of the next prs, utilities already have docs